test/e2e/framework/framework.go

```diff
@@ -16,8 +16,14 @@ package framework
 
 import (
 	"bufio"
+	"bytes"
 	"context"
 	"fmt"
+	"k8s.io/client-go/rest"
+	"k8s.io/client-go/tools/portforward"
+	"k8s.io/client-go/transport/spdy"
```

Comment:
reorder

Reply:
yes, we should add a linter, but not as a github action since we are already using prow downstream.